src/classes/Entity/types.ts

```diff
@@ -340,7 +342,7 @@ export type BaseOptions<
   Execute extends boolean | undefined = undefined,
   Parse extends boolean | undefined = undefined
 > = {
-  capacity: DocumentClient.ReturnConsumedCapacity
+  capacity: ReturnConsumedCapacity | `${ReturnConsumedCapacity}` | Lowercase<ReturnConsumedCapacity>
```

Comment:
is not it enough to have only `capacity: ReturnConsumedCapacity` ?

Reply:
basically the above allows us to do the following:

```typescript
// ....
  { metrics: 'total' } 
// OR
 { metrics: 'TOTAL' }
// OR
{capacity: ReturnConsumedCapacity.TOTAL }

this is something we already support on the implementation level so I think it's a nice to have type definition.
Wdyt?

Reply:
👍🏻